src/LowLevel/types.ts

```diff
@@ -229,6 +229,22 @@ export interface GetSecurityTokenArgs {
   ticker: string;
 }
 
+export interface GetTickerDetailsArgs {
+  ticker: string;
+}
+
+export interface IsTickerAvailableArgs {
+  ticker: string;
+}
+
+export interface TickerDetails {
```

Comment:
This interface is only used in the SecurityTokenRegistry, so it should probably be moved there. Also, try to follow the same format I've been using in the comments for each property (JSDoc style) since it plays nicer with the editor (it shows the property documentation when you're accessing it for example)